src/pages/CommonComponents/Footer.js

```diff
@@ -1,53 +1,61 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
 import './Footer.scss';
 
-export default class Footer extends Component {
+class Footer extends Component {
   render() {
+    const currentPage = this.props.location.pathname;
     return (
-      <footer className="commonFooter">
-        <section className="evaluationFooter">
-          <span>
-            지금까지 <span className="footerCounter">얼마</span>의 평가
-          </span>
-        </section>
-        <section className="descriptionFooter">
-          <div className="siteDescription">
-            <ul>
-              <li>
-                <span>이 사이트는 공식 왓챠 사이트가 아닙니다</span>
-              </li>
-              <li>
-                <span>이 사이트는 공식 왓챠 사이트와 관련이 없습니다</span>
-              </li>
-              <li>
-                <span>
-                  이 사이트는 학생들이 공부용으로 만든 클론 프로젝트
-                  사이트입니다
-                </span>
-              </li>
-            </ul>
-          </div>
-          <div className="memberDescription">
-            <ul>
-              {MEMBER_LIST.map(member => {
-                return (
-                  <li key={member.id}>
-                    <button>
-                      <a href={member.blog} target="_blank">
-                        <span>{member.name}</span>
-                      </a>
-                    </button>
+      <>
+        {currentPage !== '/review' && (
```

Comment:
전체페이지에 대한 조건부렌더링이면 하나의 return 문 안에서 처리하는 것 보다
```suggestion
        render(){
         const currentPage = this.props.location.pathname;
         if(currentPage === "review") return null
         return (
           <footer ~~~~~~~~>
```
이런식으로 처리하는게 좀 더 깔끔해 보입니다!

Reply:
null을 안쓰려고 단축평가 문법을 사용해서 조건부 렌더링을 한다고 알고 있었는데 이 경우엔 ()안에 너무 길어져서 if문으로 처리하는건가용?!?

Reply:
삼항연산자를 사용하는것보다 단축평가 문법이 좀 더 간결해서 불필요하게 삼항연산자 사용은 지양하라고 권장드리고 있습니다.

저는 개인적으로 이렇게 컴포넌트 전체의 렌더링 여부를 결정지을때는 render method안의 () 안에 다 들어가있으면 보기 지저분해보여서 그 위에서 if문 사용해서 조건 맞을때는 null return 하면서 끝내버리는게 깔끔하고 명확하다고 생각되어서 저렇게 사용하고있습니다.

단순히 가독성적인 측면이라서 어떤게 더 깔끔해보이는지 한번 생각만해보시면 될 것 같습니다. 어떤게 더 낫다 라는 건 없습니다!

Reply:
아아 무조건 null을 쓰는게 안 좋다~ 그런게 아니군요!! 확실히 이게 더 보기 좋아보여요!!! 감사합니다 👍🏻